lib/account/account-service.js

```diff
@@ -69,6 +69,26 @@ module.exports = function (config, secureHash, storage) {
 
 
     /**
+     * Delets the login file in the account directory.
+     *
+     * @param {string} hashedAccountId
+     * @param {string} loginId
+     * @return {Promise.<boolean>}
+     */
+    function deleteLoginFileAsync(hashedAccountId, loginId) {
+        return secureHash.hashAsync(loginId, accountConfig.idHash).then((hashedLoginId) => {
+            var loginKey;
+
+            loginKey = makeLoginKey(hashedAccountId, hashedLoginId);
+
+            return storage.delAsync(loginKey);
+        }).then(() => {
+            return true;
```

Comment:
Does it matter that this returns a boolean?  Can it just return a promise that resolves or does not resolve?